mesonbuild/wrap/wraptool.py

```diff
@@ -207,9 +206,4 @@ def status(options):
 
 def run(options):
     options.wrap_func(options)
-
-if __name__ == '__main__':
-    parser = argparse.ArgumentParser(prog='wraptool')
-    add_arguments(parser)
-    options = parser.parse_args(sys.argv[1:])
-    run(options)
+    return 0
```

Comment:
Why do you add this in the commit porting wraptool to argparse and then remove it here?

Reply:
Because wraptool had a __main__ that parse manually command line, so I just ported it as-is using argparse. It's only later I realized that __main__ is legacy that doesn't even work anymore, so I removed it in a few places.